### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,22 +30,22 @@ Code status
 Download
 --------
 
-You can get the newest release at https://www.phpmyadmin.net/.
+https://www.phpmyadmin.net/ always has the newest release.
```
Comment (Guileas) [inline README.rst:33]:
```suggestion
You can get the newest release at https://www.phpmyadmin.net/.
```

File: README.rst [hunk 2]
```diff
@@ -30,22 +30,22 @@ Code status
...
 
-Note that phpMyAdmin uses Composer to manage library dependencies, when using git
-development versions you must manually run Composer.
+Note that phpMyAdmin uses `Composer <https://getcomposer.org/>`_ to manage library dependencies, when using Git
+development versions, you must manually run Composer.
 Please see `the documentation <https://docs.phpmyadmin.net/en/latest/setup.html#installing-from-git>`_ for details.
 
-More Information
-----------------
+More Info
```
Comment (Guileas) [inline README.rst:45]:
```suggestion
More Information
```

**Review by Guileas — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (comradekingu) [reply README.rst:33]: Harder to copy the link.

File: README.rst (hunk 2 above)
Comment (comradekingu) [reply README.rst:45]: This is just longer for no good reason.

**Review by ibennetch — CHANGES_REQUESTED** [review]
Thanks for your submission, I have a few suggestions.

File: README.rst [hunk 3]
```diff
@@ -38,20 +42,20 @@ Download
...
 Please see `the documentation <https://docs.phpmyadmin.net/en/latest/setup.html#installing-from-git>`_ for details.
 
 More Information
 ----------------
 
-Please see the documentation in the doc folder or at https://docs.phpmyadmin.net/.
+Please see https://docs.phpmyadmin.net/, or browse the documentation in the doc folder.
 
-For support or to learn how to contribute code or by translating to your language,
+For `support <https://www.phpmyadmin.net/support/>`_ or information on contributing code or `translating phpMyAdmin to your language <https://hosted.weblate.org/projects/phpmyadmin/>`_,
```
Comment (Guileas) [inline README.rst:60]:
```suggestion
For `support <https://www.phpmyadmin.net/support/>`_ or information concerning code contribution or `translating phpMyAdmin to your language <https://hosted.weblate.org/projects/phpmyadmin/>`_,
```

Comment (comradekingu) [reply README.rst:60]: The two latter ones are more of the same type of thing, whereas the change makes it more three distinct things (?)

File: README.rst [hunk 4]
```diff
@@ -42,20 +42,23 @@ Download
...
-Please see the documentation in the doc folder or at https://docs.phpmyadmin.net/.
+Please see https://docs.phpmyadmin.net/, or browse the documentation in the doc folder.
 
-For support or to learn how to contribute code or by translating to your language,
-visit https://www.phpmyadmin.net/
+For `support <https://www.phpmyadmin.net/support/>`_ or `security issues <https://www.phpmyadmin.net/security/>`_ you can visit https://www.phpmyadmin.net/
+
+Translations are welcome, you can `translate phpMyAdmin to your language <https://hosted.weblate.org/projects/phpmyadmin/>`_.
+
+If you are into code you can read the `code contribution file <CONTRIBUTING.md>`_ or browse our website `contributing page <https://www.phpmyadmin.net/contribute/>`_.
```
Comment (shucon) [inline README.rst:64]: If you would like to contribute to the phpMyAdmin's codebase, you can read the `code contribution file <CONTRIBUTING.md>`_ or browse our website's contributing page <https://www.phpmyadmin.net/contribute/>`_.

Looks better.

Comment (williamdes) [reply README.rst]: Awesome !

**Comment by shucon** [comment]
LGTM

**Review by shucon — APPROVED** [review]

**Review by Guileas — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Pull request merged**